docs/getting-started/products/yeth/overview.md

```diff
@@ -0,0 +1,94 @@
+# Overview
+
+**yETH**
+
+yETH is a user-governed liquidity pool token consisting of various Ethereum Liquid Staking Derivatives (LSDs). yETH represents one-to-one [beacon chain ETH](https://ethereum.org/en/upgrades/beacon-chain/) (a.k.a. "ETH staked").
+
+yETH does not accrue any yield, it all goes to st-yETH, which makes yETH a good token to use in a stableswaps like Curve. In order to mint yETH users must provide liquidity for all LSDs currently in the yETH pool, or trade in the yETH<->ETH Curve pool.
+
+**Staked yETH (st-yETH)**
+
+Users can deposit yETH to mint st-yETH, and can burn st-yETH to receive yETH back from the staking contract. Stakers receive all yield and slashings from beacon chain (Ethereum proof-of-stake validators).
```

Comment:
> Users can deposit yETH to mint st-yETH, and can burn st-yETH to receive yETH back from the staking contract. Stakers receive all yield and slashings from beacon chain (Ethereum proof-of-stake validators).

Users stake their yETH to mint st-yETH, accrue yield, and later unstake st-yETH to receive yETH back according to their earnings. Stakers receive all yield and slashings from beacon chain (Ethereum proof-of-stake validators) and earn incentives if they participate and vote in yETH governance.

Reply:
```suggestion
Users stake their yETH to mint st-yETH, accrue yield, and later unstake st-yETH to receive yETH back according to their earnings. Stakers receive all yield and slashings from beacon chain (Ethereum proof-of-stake validators) and earn incentives if they participate and vote in yETH governance.
```